pkg/httphelper/httphelper.go

```diff
@@ -0,0 +1,57 @@
+package httphelper
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+// shamelessly taken from martini-contrib/render/render.go
+const (
+	ContentType = "Content-Type"
+	ContentJSON = "application/json"
+)
+
+type ResponseHelper interface {
+	Error(error)
+	JSON(int, interface{})
+	WriteHeader(int)
+}
```

Comment:
We have this same interface [in the controller](https://github.com/flynn/flynn/blob/master/controller/controller.go#L88-L113), can it be shared?